### PR TITLE
Support mkr plugin install

### PR DESCRIPTION
**Pull request by syou6162**
Thank you for providing useful mackerel plugin! Recently, plugin users can install third party's plugins (like this plugin) with [mkr](https://github.com/mackerelio/mkr).

- https://mackerel.io/ja/docs/entry/advanced/install-plugin-by-mkr
- https://mackerel.io/ja/docs/entry/advanced/make-plugin-corresponding-to-installer

This pull request enables us to install this plugin via `mkr plugin install tjinjin/mackerel-plugin-aws-billing`.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package main
+
+import mpawsbilling "github.com/syou6162/check-docker-ps/lib"
```
Comment (tjinjin) [inline main.go]: Typo? 🤔 　`github.com/tjinjin/mackerel-plugin-aws-billing/lib`

**Comment by tjinjin** [comment]
Thank you for your PR :tada:
If fix bug, I'll merge soon!

File: main.go (hunk 1 above)
Comment (syou6162) [reply main.go]: exactly 🙇  I fixed it!

**Comment by syou6162** [comment]
Thank you for your quick response!

After you merged, I would like to ask you to do following things:

- `git tag v0.0.2` for merged master branch
  - you already have v0.0.1 release https://github.com/tjinjin/mackerel-plugin-aws-billing/releases/tag/v0.0.1
- `GITHUB_TOKEN=... script/release`
- ref: https://mackerel.io/ja/docs/entry/advanced/make-plugin-corresponding-to-installer

After the above processes, we can finally do `mkr plugin install tjinjin/mackerel-plugin-aws-billing` 🎉

**Comment by tjinjin** [comment]
release v0.0.2 :tada:

**Comment by syou6162** [comment]
Thank you for merging & releasing!

**Pull request merged**